gdal/frmts/tiledb/tiledbdataset.cpp

```diff
@@ -929,10 +933,18 @@ GDALDataset *TileDBDataset::Open( GDALOpenInfo * poOpenInfo )
         {
             if ( dims.size() == 3 )
             {
-                poDS->nBands = dims[0].domain<size_t>().second
-                                - dims[0].domain<size_t>().first + 1;
-                poDS->nBlockYSize = dims[1].tile_extent<size_t>();
-                poDS->nBlockXSize = dims[2].tile_extent<size_t>();
+                if ( ( pszAttr != nullptr ) && ( schema.attributes().count( pszAttr ) == 0 ) )
```

Comment:
that one too

Reply:
ok, thanks @rouault I have not done one of these before, I will make the changes